### ISSUE TITLE
Is there still a need for more p5.shader examples?

### ISSUE THREAD

**Issue by thantr** [processing/p5.js#6134]
### Increasing Access

As there already is a Mandelbrot set, I would love to work on and provide a webGL example that showcases a Julia set fractal 

### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [X] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### Feature enhancement details

I am asking to see if there is any interest in a webGL live julia set example as there is a Mandelbrot set example. I would love to work on this please let me know as i would love to provide something like this

**Comment by davepagurek** [comment]
I think the Mandelbrot example we currently have is in the documentation for `loadShader`, but if one looks in the 3D section of the [examples page](https://p5js.org/examples/), there aren't any fractal examples, so a Julia set would be great!

One thing I notice with the existing shader examples there is that if they use `loadShader`, the shader code isn't actually visible on the example page. Maybe it's worth using `createShader` and passing in the code as strings so that people can see the shader source on that page?

Also if you are interested in making a new capital-e Example (rather than an example directly in the reference for a function), those live here https://github.com/processing/p5.js-website/tree/main/src/data/examples/en/20_3D in the p5.js-website repo. If you make a PR there, feel free to tag me 🙂

**Comment by thantr** [comment]
Yes, I think having the shader code be visible would be a great idea! Thank you

**Comment by davepagurek** [comment]
That's this one right? https://github.com/processing/p5.js-website/pull/1364 I've tagged myself for review in there now, I'll take a look soon!

**Comment by thantr** [comment]
yes it is ! im eager to hear back from you
